inception-project/inception
# #3492 - Deleting a layer feature doesn't always work

**What's in the PR**

* Move the "show" call after adding the lambda because debugging showed that the "confirmaAction" in the Dialog was "null" on the first call to it. While the change is not the most logical solution, it empirically removes the bug.

**How to test manually**
* Create a project with document layer
* See that it only needs one deletion of the layer where before it needed two

**Automatic testing**

**Documentation**
